.drone.star

```diff
@@ -2966,16 +2966,16 @@ def skipIfUnchanged(ctx, type):
             "^__mocks__/.*",
             "^packages/.*/tests/.*",
             "^tests/integration/.*",
-            "^tests/smoke/.*",
+            "^tests/e2e/.*",
```

Comment:
Note: this is the only real change. It should make the detection of changes to combinations of e2e and acceptance tests correct again.

The other changes are just names, they do not effect the actual functionality of the drone starlark script.